lib/tasks/lint.rake

```diff
@@ -1,4 +1,4 @@
-desc "Run govuk-lint with similar params to CI"
```

Comment:
We don't need this file at all. `bundle exec rubocop` is fewer keystrokes than `bundle exec rake lint`. Basically, we want to move away from wrapper scripts and now we can run `rubocop` directly we don't need to wrap things and obscure the parameters. More context is in https://trello.com/c/IEUuFOVa/129-remove-the-lint-rake-task-from-apps-that-have-it (the developer tooling group board).